### DISCUSSION TITLE
rediscover node home assistant

### DISCUSSION THREAD

**Discussion by daschmidt1994** [zwave-js/zwave-js-ui#3058] [Q&A]
Moved now from zwavejs to zwavejsui. The nodes are all back online in home assistnat.
Only the name was nodeid...
Now I changed the name and rediscovered. But only some entities are renamed.
This are my settings.
[https://postimg.cc/gallery/FYZzcFr](url)

**Comment by robertsLando** [comment]
Hi @daschmidt1994, I see you are using MQTT Discovery, in case of home assistant the best would be to use hass integration and enable zwavejs server in settings (enable `WS Server` switch [here](https://postimg.cc/mt7pnMss).

For more info check: https://zwave-js.github.io/zwave-js-ui/#/homeassistant/homeassistant-official
